fxa/oauth.py

```diff
@@ -159,21 +167,65 @@ def verify_token(self, token, scope=None):
         :raises fxa.errors.ClientError: if the provided token is invalid.
         :raises fxa.errors.TrustError: if the token scopes do not match.
         """
-        url = '/verify'
-        body = {
-            'token': token
-        }
-        resp = self.apiclient.post(url, body)
+        key = 'fxa.oauth.verify_token:%s:%s' % (token, scope)
```

Comment:
Since these are bearer tokens, I think we should hash this key so that we don't have the raw token kicking around in e.g. redis or memcache or whatever consumers want to plug in here.